docs/tools/dotnet-embedding/errors.md

```diff
@@ -22,59 +22,59 @@ E.g. parameters, environment
+* All version information
 
-The easiest way to get exact version information is to use the **Xamarin Studio** menu, **About Xamarin Studio** item, **Show Details** button and copy/paste the version informations (you can use the **Copy Information** button).
+The easiest way to get exact version information is to use the **Xamarin Studio** menu, **About Xamarin Studio** item, **Show Details** button and then copy/paste the version information (you can use the **Copy Information** button).
 
 <a name="EM0001" />
 
 ### EM0001: Could not create Output directory `X`
 
-The directory name specified by `-o=DIR` does not exists and could not be created. It might be an invalid name for the file system.
+The directory name specified by `-o=DIR` does not exist and could not be created. It might be an invalid name for the file system.
 
 <a name="EM0002" />
 
 ### EM0002: Option `X` is not supported
 
-The tool does not support the option `X`. It is possible that another version of the tool supports it or that it does not apply in this environment.
+The tool does not support the option `X`. It is possible that another version of the tool supports it or the `X` option does not apply in this environment.
 
 <a name="EM0003" />
 
 ### EM0003: The platform `X` is not valid.
 
-The tool does not support the platform `X`. It is possible that another version of the tool supports it or that it does not apply in this environment.
+The tool does not support the platform `X`. It is possible that another version of the tool supports it or the `X` platform does not apply in this environment.
 
 <a name="EM0004" />
 
 ### EM0004: The target `X` is not valid.
 
-The tool does not support the target `X`. It is possible that another version of the tool supports it or that it does not apply in this environment.
+The tool does not support the target `X`. It is possible that another version of the tool supports it or the `X` target does not apply in this environment.
 
 <a name="EM0005" />
 
 ### EM0005: The compilation target `X` is not valid.
 
-The tool does not support the compilation target `X`. It is possible that another version of the tool supports it or that it does not apply in this environment.
+The tool does not support the compilation target `X`. It is possible that another version of the tool supports it or the `X` compilation target does not apply in this environment.
 
 <a name="EM0006" />
 
 ### EM0006: Could not find the Xcode location.
 
-The tool could not find the currently selected Xcode location using the `xcode-select -p` command. Please verify that this command succeeds, and returns the correct Xcode location.
+The tool could not find the currently selected Xcode location using the `xcode-select -p` command. Please verify that you can successfully run this command and it returns the correct Xcode location.
 
 <a name="EM0007" />
 
-### EM0007: Could not get the sdk version for '{sdk}'.
+### EM0007: Could not get the sdk version for `{sdk}`.
```

Comment:
Ideally these strings match exactly what is emitted in the error console. I need to verify whether apostrophes or backticks are used in the product. Same applies to all edited headings in this PR.

Reply:
Yeah, I wasn't sure either. If the emitted strings include the single quotes, I would advocate for putting the single quotes back in and keep the back ticks to denote the placeholder. This would keep it consistent with both the emitted output and the placeholder formatting in other headings.

Reply:
The source shows that apostrophes are correct:

```
throw ErrorHelper.CreateError (7, $"Could not get the sdk version for '{sdk}'");
```